BondageClub/Scripts/Dialog.js

```diff
@@ -126,6 +126,17 @@ var DialogSelfMenuOptions = [
 	},
 ];
 
+/**
+ * Returns character based on argument
+ * @param {Character | string} C - The characer to get; can be `"Player"` to get player or empty to get current
+ * @returns {Character} - The actual character
+ */
+function DialogGetCharacter(C) {
+	if (typeof C === 'string')
```

Comment:
I know it's not in the lint rules (yet), but should probably aim for quote consistency (and double seems to be the overwhelming majority in this particular file).
```suggestion
	if (typeof C === "string")
```